chainer/functions/pooling/unpooling_2d.py

```diff
@@ -89,7 +89,7 @@ def unpooling_2d(x, ksize, stride=None, pad=0, outsize=None, cover_all=True):
             :func:`~chainer.utils.conv.get_conv_outsize` must be equal to
             the shape of the 2d array in the input batch ``x``.
         cover_all (bool): If ``True``, all spatial locations are pooled
```

Comment:
cover_all (bool): If ``True``, the output size may be smaller than if cover_all is ``False``. This flag serves to align behavior to the pooling functions which can cover all input locations, see :func:`max_pooling_2d`, :func:`convolution_2d`